datadog/resource_datadog_synthetics_test_test.go

```diff
@@ -263,6 +263,23 @@ func TestAccDatadogSyntheticsTCPTest_Basic(t *testing.T) {
 	})
 }
 
+func TestAccDatadogSyntheticsTCPLegacyOptionsTest_Basic(t *testing.T) {
```

Comment:
Does this test give us anything specific? 
The other two look to cover the case of 
* ensuring we can still set the old options 
* ensuring that if you upgrade from old -> new, the old is no longer present

Reply:
Good point, updated a test instead. We do cover the 2 point.